caravel/models.py

```diff
@@ -17,6 +17,7 @@
 import requests
 import sqlalchemy as sqla
 import sqlparse
+import jinja2
```

Comment:
We'll may be more consistent but we'll be less robust. Also i don't want to handle a stream of github issues because people have borked installations :) If we don't require any specific jinja2 feature / bugfix, the version we get from flask / fab / whatever would be fine.